.changeset/big-spoons-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': minor
+---
+
+Refactors registry structure by moving filesystem components to dedicated directory and adds ESLint restrictions to prevent Node.js imports in browser code.
```

Comment:
A changeset description reads better if it says what have changed, not what will change. As in "this thing was improved" rather than "changes this thing". For example:

> The registry code is restructured by moving filesystem components to a dedicated directory. ESLint restrictions added to prevent Node.js imports in browser components.